marlowe-playground-client/src/MainFrame/State.purs

```diff
@@ -485,6 +518,7 @@ handleAction (NewProjectAction (NewProject.CreateProject lang)) = do
 handleAction (NewProjectAction NewProject.Cancel) = fullHandleAction CloseModal
 
 handleAction (DemosAction action@(Demos.LoadDemo lang (Demos.Demo key))) = do
+  liftEffect $ SessionStorage.setItem metadataLocalStorageKey (encodeJSON (emptyContractMetadata :: MetaData)) -- ToDo: Load metadata for example (SCP-1912)
```

Comment:
Not sure how you wanted to implement SCP-1912, but I think it makes sense to remove the `Examples.*` that are based on string, and start using the `Marlowe.Market.*` defined in web-common-marlowe that already have the metadata.

We should probably add there the JS and HS examples. This way we would have a sync between the dashboard and the playground.

Reply:
In any case, if you load the metadata (either empty or prepopulated) to the State and use the Session, then this `setItem` would not be needed.